app/grandchallenge/cases/views.py

```diff
@@ -94,6 +101,26 @@ class RawImageUploadSessionViewSet(
 ):
     serializer_class = RawImageUploadSessionSerializer
     queryset = RawImageUploadSession.objects.all()
+    permission_classes = [DjangoObjectOnlyWithCustomPostPermissions]
+    filter_backends = [ObjectPermissionsFilter]
 
     def perform_create(self, serializer):
         serializer.save(creator=self.request.user)
+
+    @action(detail=True, methods=["patch"])
+    def process_images(self, request, pk=None):
+        upload_session = self.get_object()
+        upload_session.process_images()
```

Comment:
We should only schedule this job if there are unprocessed images, and if the `RawImageUploadSession.session_state` is `UploadSessionState.stopped`, otherwise users could flood the system with jobs.

Reply:
@jmsmkn 
Isnt checking uplaod_session.session_state == UploadSessionState.stopped enough? We can add a field to RawImageFile model which will be marked processed? 
```
im.state == "processed" for all im in RawImageFile.objects.filter(upload_session=upload_session).all()
```

Reply:
Ok, yes, it turns out that state is never saved. We need a migration for that, but that can be another issue.